automod/conditions.go

```diff
@@ -318,7 +318,7 @@ func (cd *ChannelCategoriesCondition) MergeDuplicates(data []interface{}) interf
 /////////////////////////////////////////////////////////////////
 
 type AccountAgeConditionData struct {
-	Treshold int
+	Threshold int
```

Comment:
We cannot rename effect/trigger data fields; they are serialized and saved to database so existing data would no longer deserialize correctly after this change. Please revert.